RealmSwift/Object.swift

```diff
@@ -250,6 +250,16 @@ public final class DynamicObject : Object {
         }
         return nil
     }
+
+    /// :nodoc:
+    public override func valueForUndefinedKey(key: String) -> AnyObject? {
```

Comment:
It was either performance or so that KVC accessors for other properties would continue to work